pkg/reporter/format.go

```diff
@@ -0,0 +1,41 @@
+package reporter
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"golang.org/x/term"
+)
+
+var format = []string{"table", "json", "markdown", "sarif"}
+
+func Format() []string {
+	return format
+}
+
+func GetReporter(format string, stdout, stderr io.Writer, fileOutput bool) (Reporter, error) {
```

Comment:
nit: avoid stutter here by just calling this `Get`. i.e. `reporter.Get`. Or better just `reporter.New`.